src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
         });
       } else {
         return child;
       }
     },
 
+    _hasSelectedDescendant(previousValue, child) {
+      if (child.props.nestedItems.length > 0) {
```

Comment:
Alternatively you can simply return false if `!React.isValidElement(child)` I mean:

```js
_hasSelectedDescendant(previousValue, child) {
  if (!React.isValidElement(child)) return false;
  if (&& child.props.nestedItems && child.props.nestedItems.length > 0) {
    //...
  }
}
```